typo.go

```diff
@@ -4,7 +4,9 @@
 
 // Typo is a modern version of the original Unix typo command, a scan of whose man page
 // is at
-//   https://github.com/robpike/typo/blob/master/typo.png
+//
```

Comment:
why double this?

Reply:
ditto.

Reply:
really? one before and two after? well, if this is gofmt output, ok.

Reply:
I don't make the rules. :)